src/runtime/hexagon_remote/halide_host.cpp

```diff
@@ -237,13 +237,24 @@ void halide_hexagon_host_free(void *ptr) {
 
     free(rec);
 }
-
 // This is a shim for calling v2 from v1.
 handle_t halide_hexagon_remote_get_symbol(handle_t module_ptr,
                                           const char* name, int nameLen) {
     handle_t sym = 0;
-    int result = halide_hexagon_remote_get_symbol_v2(module_ptr, name, nameLen, &sym);
+    int result = halide_hexagon_remote_get_symbol_v2(module_ptr, name, nameLen, false, &sym);
     return result == 0 ? sym : 0;
 }
 
+handle_t halide_hexagon_remote_get_symbol_shared_object(handle_t module_ptr,
```

Comment:
This particular function name" halide_hexagon_remote_get_symbol_v2"  is already in existence in the patch.
perhaps we could rename it to be "halide_hexagon_remote_get_symbol_handle ?